app/views/admin/videos/1new.html.haml

```diff
@@ -0,0 +1,38 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], html: { class: 'form-horizontal' } do |f|

```

Comment:
`html: { class: 'form-horizontal' }` is the old style, which no longer works. You need to use `layout: :horizontal` now. Once you do that you don't have to create your form groups and columns manually below.